Source/Core/DolphinQt/Config/GameConfigWidget.cpp

```diff
@@ -102,18 +103,19 @@ void GameConfigWidget::CreateWidgets()
                                "games. (ON = Compatible, OFF = Fast)"));
   m_sync_gpu->setToolTip(tr("Synchronizes the GPU and CPU threads to help prevent random freezes "
                             "in Dual core mode. (ON = Compatible, OFF = Fast)"));
-  m_enable_fast_disc->setToolTip(tr("Enable emulated disc speed. Disabling this can cause crashes "
+  m_emulate_disc_speed->setToolTip(tr("Enable emulated disc speed. Disabling this can cause crashes "
                                     "and other problems in some games. "
                                     "(ON = Compatible, OFF = Unlocked)"));
```

Comment:
It looks like this now exceeds the line length limit and the linter wants it changed. This would probably the nicest-looking change:

```c++
  m_emulate_disc_speed->setToolTip(tr("Enable emulated disc speed. Disabling this can cause "
                                      "crashes and other problems in some games. "
                                      "(ON = Compatible, OFF = Unlocked)"));
```

(the linter itself doesn't like breaking up strings and makes a different suggestion.)

Reply:
`clang-format`ted it.